CMake/itkExternal_Eigen3.cmake

```
@@ -13,57 +17,58 @@ mark_as_advanced(ITK_USE_EIGEN_MPL2_ONLY)
 if(ITK_USE_EIGEN_MPL2_ONLY)
   list(APPEND _additional_external_project_args "-DITK_USE_EIGEN_MPL2_ONLY:BOOL=ON")
 endif()
+option(ITK_EIGEN_MANGLE "Replace Eigen namespace with itkeigen to avoid conflicts with other Eigen versions." OFF)
```

```suggestion
option(ITK_EIGEN_MANGLE_INCLUDE_DIR "To avoid conflicts with other Eigen versions, expect itkeigen/Eigen include directory instead of Eigen." ON)
```

Notes:

* enabled by default to match existing behavior.

---

MANGLE is discarded